config/kafka-monitor.properties

```diff
@@ -67,7 +67,10 @@
       "kmf.services:type=consume-service,name=*:records-delay-ms-avg",
       "kmf.services:type=produce-service,name=*:produce-error-rate",
       "kmf.services:type=consume-service,name=*:consume-error-rate"
-    ]
+    ],
+    "report.graphite.host": "localhost",
```

Comment:
The `kafka-monitor.properties` is supposed to be used as a quick-start example for user to try kafka-monitor. Ideally we want to keep it simple and include the most commonly used configs. I am not sure most user will use `graphte`. Maybe we can leave it out of the default config here?